pkg/libfixmath/Makefile

```diff
@@ -1,7 +1,7 @@
 PKG_NAME     := libfixmath
 PKG_VERSION  := ad9ed940e57d43432b276e95aee6ca9df6f23ccd
-PKG_URL      := git://github.com/PetteriAimonen/libfixmath
-PKG_BUILDDIR ?= $(BINDIRBASE)/pkg/$(BOARD)/$(PKG_NAME)
+PKG_URL      := https://github.com/PetteriAimonen/libfixmath
```

Comment:
There is a new Git over HTTPS protocol (smart HTTP) that GitHub provides that is similar to the Git protocol and SSH [[1]]. Since non-HTTP(S) protocols are sometimes blocked by firewalls, I think it's better to use HTTPS from now on.

[1]: https://git-scm.com/book/en/v2/Git-on-the-Server-The-Protocols#The-HTTP-Protocols